its/src/test/java/com/sonar/it/csharp/UnitTestProjectTypeProbingTest.java

```diff
@@ -73,8 +73,10 @@ public void testProject_WithPropertySetToFalse_IsIdentifiedAsMainProjectByScanne
   }
 
   @Test
-  public void msTestProject_IsIdentifiedAsTestProject() {
-    assertThat(getIssues("UTProjectProbing:UTProjectProbing.MsTest/calculator.cs")).isEmpty();
+  public void msTestProject_IsIdentifiedAsMainProjectBzScannerAndTestByAnalyzer() {
```

Comment:
nitpick
msTestProject_IsIdentifiedAsMainProjectByScannerAndTestByAnalyzer
(not BzScanner)